features/step_definitions/money_manager_steps.rb

```diff
@@ -32,3 +44,7 @@
   expect(money_manager_circumstances_changed_page.received_first_payment_false).to be_checked
   expect(money_manager_circumstances_changed_page.single_or_in_couple_couple).to be_checked
 end
+
+Given(/^I get my results for the different country$/) do
```

Comment:
Is it a "Then" here rather than "Given"?

Reply:
Btw, the description says: "I get my results for the different country" but the expectation only verifies that the user is on the summary page. Do you think it is worth a renaming of the step or changing the expectation?

Reply:
Agreed. I've been looking at improving that scenario and the previous one, but there's nothing on the page that is country specific. I considered adding a step to click on one of the advice guides but that makes the test very specific because you would need to know which answers produce guides that have different text, wdyt?